canonical/lxd-ui
# use remote image json file for images, for ubuntu update local json file

## Done

- update canonical image server index
- get other images directly from server

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - instances > create > browse images
    - ensure the list contains images from canonical (ubuntu) and other distributions, and the images are available.
